controller/api/public/stat_summary.go

```diff
@@ -422,11 +425,44 @@ func (s *grpcServer) getMeshedPodCount(obj runtime.Object) (*podCount, error) {
 				meshCount.inMesh++
 			}
 		}
-	}
 
+		errors := checkContainerErrors(pod.Status.ContainerStatuses, "conduit-proxy")
+		errors = append(errors, checkContainerErrors(pod.Status.InitContainerStatuses, "conduit-init")...)
+
+		if len(errors) > 0 {
+			podErrors[pod.Name] = &pb.PodErrors{Errors: errors}
+		}
+	}
+	meshCount.errors = podErrors
 	return meshCount, nil
 }
 
+func toPodError(message string) *pb.PodErrors_PodError {
+	return &pb.PodErrors_PodError{
+		Error: &pb.PodErrors_PodError_Unknown_{
+			Unknown: &pb.PodErrors_PodError_Unknown{Message: message},
+		},
+	}
+}
+
+func checkContainerErrors(containerStatuses []apiv1.ContainerStatus, containerName string) []*pb.PodErrors_PodError {
+	errors := []*pb.PodErrors_PodError{}
+	for _, st := range containerStatuses {
+		if st.Name == containerName && st.State.Waiting != nil {
+			errors = append(errors, toPodError(fmt.Sprintf("[%s] container has not started. %s", st.Name, st.State.Waiting.Message)))
+
+			if st.LastTerminationState.Waiting != nil {
+				errors = append(errors, toPodError(fmt.Sprintf("[%s] [image: %s] %s", st.Name, st.Image, st.LastTerminationState.Waiting.Message)))
+			}
+
+			if st.LastTerminationState.Terminated != nil {
+				errors = append(errors, toPodError(fmt.Sprintf("[%s] [image: %s] %s", st.Name, st.Image, st.LastTerminationState.Terminated.Message)))
```

Comment:
what do you think about making each of these case its own PodError type, rather than making them all `Unknown` that way, we could have an `image` field when appropriate, for instance, rather than an encoded string?

Reply:
Also, it's probably redundant to include the pod name in the message, since it's a key in the pod error map

Reply:
ooh. yeah, it would be nice to not have to encode all this information in the string. what about:

```
    message ContainerError {
      string message = 1;
      string container = 2;
      string image = 3;
    }
```

Reply:
`st.Name` is the container name, not the pod name; but if we have a container error type, we could have a field for name too (I named it `container` above, but it could be `name`)